nomt/src/store.rs

```diff
@@ -112,8 +115,12 @@ impl Transaction {
     pub fn write_page<V: AsRef<[u8]>>(&mut self, page_id: PageId, value: Option<V>) {
         let cf = self.shared.db.cf_handle(PAGES_CF).unwrap();
         match value {
-            None => self.batch.delete_cf(&cf, page_id.to_bytes().as_ref()),
-            Some(value) => self.batch.put_cf(&cf, page_id.to_bytes().as_ref(), value),
+            None => self
+                .batch
+                .delete_cf(&cf, page_id.length_dependent_encoding()),
+            Some(value) => self
+                .batch
+                .put_cf(&cf, page_id.length_dependent_encoding(), value),
```

Comment:
Wouldn't it be more beneficial if this was length dependent **and** packed. Otherwise limb wastes 2 bits.

Reply:
I am not sure how to do that right now. We would have to chop off leading bits rather than bytes.